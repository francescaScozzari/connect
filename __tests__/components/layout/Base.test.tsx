import { expect } from '@jest/globals'
import React from 'react'

import Layout from '@/components/layout/Base'
import { renderWithWrappers } from '@/__tests__/functions'

const setup = () =>
  renderWithWrappers(
    <Layout>
      <h1>Lorem Ipsum</h1>
    </Layout>
  )

describe('Layout components', () => {
  describe('<Layout />', () => {
    test('Primary layout renders correctly', () => {
      const container = setup()
      expect(container.firstChild).toMatchSnapshot()
    })
  })
})
