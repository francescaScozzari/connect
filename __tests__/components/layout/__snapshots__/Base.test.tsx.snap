// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout components <Layout /> Primary layout renders correctly 1`] = `
<h1>
  Lorem Ipsum
</h1>
`;
